ASReviewLAB.qmd

```diff
@@ -161,12 +143,19 @@ When performing a systematic review with your own data, you need to
 provide the prior knowledge yourself (at least one relevant and one
 irrelevant record). However, because you are using the Validation Mode 
 of ASReview, the relevant records are known; the original authors have 
-already read ALL records.
-
-To select the prior knowledge you first need to click on the ‘`ADD`’
-button next to ‘`Add prior knowledge`’; see also the documentation 
+already read ALL records. See also the documentation 
 about the selection of [prior knowledge](https://asreview.readthedocs.io/en/latest/lab/project_create.html#prior-knowledge){target="_blank"}.
-Now you will see a menu about selecting prior knowledge.
+
+To select the prior knowledge you first need to click on the `Customize`
```

Comment:
In the previous step you already clicked on "Screen" so the user is in the reviewer screen. If you want them to add prior knowlegde, you should change the previous step.